README.md

```diff
@@ -179,6 +164,46 @@ MKdocs will generate static HTML files for Chainsafe markdown files located in `
 
 `make mkdocs`: Run MkDoc's docker image, building and hosting the html files on `localhost:8000`  
 
+# Local run
+## Prepare:
+1. Build executable:
+```
```

Comment:
Please, add a tab space before all content under the numbered list items so it looks visually as inner